eventstore/migrations/0018_auto_20200129_1126.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2020-01-29 11:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("eventstore", "0017_pmtctregistration")]
+
+    operations = [
+        migrations.AlterField(
+            model_name="pmtctregistration",
+            name="pmtct_risk",
+            field=models.CharField(
+                choices=[("Normal", "Normal"), ("High", "High")], max_length=6
```

Comment:
We'll probably need a new migration to apply the changes made to the model. If possible, could we delete all these new migrations, and just make it a single migration for this PR